bin/exec.go

```diff
@@ -0,0 +1,170 @@
+package main
+
+import (
+	"flag"
+	"github.com/litmuschaos/chaos-executor/pkg/utils"
+	"github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
+	log "github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
```

Comment:
This can be as,
```
	"flag"

	log "github.com/sirupsen/logrus"
	corev1 "k8s.io/api/core/v1"
	"k8s.io/client-go/rest"
	"k8s.io/client-go/tools/clientcmd"

	"github.com/litmuschaos/chaos-executor/pkg/utils"
	"github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
```